source/backtraceMetric.ts

```diff
@@ -0,0 +1,226 @@
+import { BacktraceClientOptions } from '.';
+import { uuid } from './utils';
+
+declare const __VERSION__: string;
+
+/**
+ * Function to create instance of BacktraceMetric and ping.
+ */
+export default function backtraceMetric(
+  configuration: BacktraceClientOptions,
+): void {
+  new BacktraceMetric(configuration).ping();
+}
+
+/**
+ * Handles Backtrace Metrics.
+ * TODO: Consider converting to a static class since an instance is only created to call a single function once.
+ */
+export class BacktraceMetric {
+  public universe;
+  public token;
+  public timeout = 15000; // fifteen seconds in miliseconds
+
+  /** Seconds since epoch. */
+  public readonly CURRENT_TIMESTAMP: number = Math.floor(
+    new Date().getTime() / 1000,
+  );
+  /** Thirty minutes in seconds. */
+  public readonly THIRTY_MINUTES = 1800;
```

Comment:
Overall the THIRTY_MINUTES variable doesn't tell me too much - why do we need it here? I would suggest to rename it to something different nextSubmission

Reply:
I renamed to persistenceInterval, which has a default value of 30 minutes. If that value is available in the client options we could update it in the constructor